docs/learn-consensus.md

```diff
@@ -52,15 +70,16 @@ The [Rust implementation](https://github.com/paritytech/substrate/blob/master/sr
 
 For even more detail, see the [GRANDPA research page](http://research.web3.foundation/en/latest/polkadot/GRANDPA/) on the W3F Research pages.
 
-### BABE: Block Production
+### Fork Choice
 
-BABE (Blind Assignment for Blockchain Extension) is the block production mechanism that runs between the validator nodes and determines the authors of new blocks.
+Bringing BABE and GRANDPA together, the fork choice of Polkadot becomes clear. When there are competing forks, the following is how priority is determined for both BABE to produce new blocks on top of, and for GRANDPA to finalize:
```

Comment:
I'm not crazy about this section. It's not wrong per se, but I think a little misguided. One of the requirements of using GRANDPA is that whatever block production engine is building on it MUST follow the fork choice rule of GRANDPA.
```suggestion
Bringing BABE and GRANDPA together, the fork choice of Polkadot becomes clear. BABE must always build on the chain that has been finalized by GRANDPA. When there are forks after the finalized head, BABE provides probabilistic finality by building on the chain with the most primary blocks.
```